api/src/main/java/org/pmiops/workbench/mail/MailService.java

```diff
@@ -3,8 +3,13 @@
 import javax.mail.Message;
 import javax.mail.MessagingException;
 
+import com.google.api.services.admin.directory.model.User;
+import org.pmiops.workbench.mandrill.model.MandrillMessage;
+import org.pmiops.workbench.mandrill.model.MandrillMessageStatuses;
+
 public interface MailService {
 
     void send(Message msg) throws MessagingException;
 
+    MandrillMessageStatuses sendEmail(String contactEmail, String password, User user) throws MessagingException;
```

Comment:
I don't understand why we're returning `MandrillMessageStatuses` since it isn't used anywhere. If we were using it for some reason, I could see returning it. As the PR stands now, it's just cruft.